asciidoc/courses/llm-fundamentals/modules/2-vectors-semantic-search/lessons/1-semantic-search/questions/2-vector-role.adoc

```diff
@@ -0,0 +1,22 @@
+[.question]
+= 2. Role of Vectors in Semantic Search
+
+What role do vectors play in semantic search systems?
+
+* [ ] They determine which advertisements to show alongside results.
+* [ ] They help in adjusting the visual appearance of search results.
+* [ ] They predict the likelihood of a user clicking on a result.
+* [*] They represent words or phrases numerically, aiding in understanding user intent.
```

Comment:
not just words and phrases in multi-modal they also represent images, audio, topology and more